### PR TITLE
Feat/updated examples

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: web-example/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (mike-parkhill) [inline web-example/README.md]: should we customize this readme?

File: web-example/craco.config.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      });
+
+      webpackConfig.module.rules.forEach((rule) => {
+        if (rule.oneOf) {
+          rule.oneOf.forEach((oneOf) => {
+            if (oneOf.loader && oneOf.loader.includes('file-loader')) {
+              oneOf.exclude.push(wasmExtensionRegExp);
+            }
+            if(oneOf.loader && oneOf.loader.includes('babel-loader') && oneOf.include) {
+              // oneOf.include = [
```
Comment (mike-parkhill) [inline web-example/craco.config.js]: what's up with this commented code?

File: web-example/public/logo192.png
Comment (mike-parkhill) [inline web-example/public/logo192.png:1]: do we need the React logos?

File: web-example/shims/react-native-sqlite-stoarge.js
Comment (mike-parkhill) [inline web-example/shims/react-native-sqlite-stoarge.js:1]: what's expected to happen with these empty files?

File: web-example/src/App.test.js
Comment (mike-parkhill) [inline web-example/src/App.test.js:1]: do we need this?

File: web-example/src/sdk-test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        name: 'test',
+    });
+
+    console.log(account);
+
+    // manage DIDs
+
+    const documents = await wallet.getAllDocuments();
+
+    console.log(documents);
```
Comment (mike-parkhill) [inline web-example/src/sdk-test.js:33]: documents could contain more than just DIDs, no?

File: web-example/craco.config.js (hunk 2 above)
Comment (maycon-mello) [reply web-example/craco.config.js]: let me remove it

File: web-example/README.md (hunk 1 above)
Comment (maycon-mello) [reply web-example/README.md]: yes, I've added a new readme

File: web-example/src/sdk-test.js (hunk 3 above)
Comment (maycon-mello) [reply web-example/src/sdk-test.js:33]: Let me copy the same example from node-js example

File: web-example/src/App.test.js
Comment (maycon-mello) [reply web-example/src/App.test.js:1]: I've removed this

File: web-example/public/logo192.png
Comment (maycon-mello) [reply web-example/public/logo192.png:1]: it was removed

File: web-example/README.md
Comment (mike-parkhill) [inline web-example/README.md:1]: should we link to the flutter and node-js  readmes from here to help people find them?

**Review by cykoder — APPROVED** [review]

**Pull request merged**